syntax = "proto3";
service Showtime {
  rpc Getshowmovies(Date) returns (ShowTimeData) {}
  rpc GetListShowtimes(Empty) returns (stream ShowTimeData) {}
}
message Date {
  string date = 1;
}

message ShowTimeData {
  string date = 1;
  repeated string idMovies = 2;
}

message Empty {
}